cloud/azure/templates/azure_saml_ses/main.tf

```diff
@@ -77,13 +67,13 @@ module "saml_keystore" {
   resource_group_name          = var.azure_resource_group
 }
 
-module "email_service" {
-  for_each = toset([
-    var.sender_email_address,
-    var.staging_applicant_notification_mailing_list,
-    var.staging_ti_notification_mailing_list,
-    var.staging_program_admin_notification_mailing_list
-  ])
-  source               = "../../../aws/modules/ses"
-  sender_email_address = each.key
-}
+# module "email_service" {
```

Comment:
**suggestion:** Could you include a comment that we're removing this because of the reliance on AWS, and it will be replaced with an alternative later? I'll create a bug too we can link here